frontend/src/component/AddIcon/AddIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  SafeAreaView,
+  StyleSheet,
+  Dimensions,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+import React from 'react';
+
+const HEIGHT = Dimensions.get('window').height;
+
+const AddIcon = () => {
+  return (
+    <SafeAreaView>
+      <TouchableOpacity >
```

Comment:
* There will be a **FlatList** (Or **ListView**) With a list of articles, so you have to make the position of the Icon absolute.